WebDriverAgentLib/Routing/FBElementUtils.h

```diff
@@ -31,11 +39,12 @@ NS_ASSUME_NONNULL_BEGIN
 + (NSSet<NSNumber *> *)uniqueElementTypesWithElements:(NSArray<id<FBElement>> *)elements;
 
 /**
- Returns all properties of FBElement protocol having 'wd' prefix
+ Returns all properties of FBElement protocol having 'wd' prefix with their getters
  
- @return list of matching property names sorted by declaration order
+ @return dictionary of matching property names with their getter names as values if getter method exists
+ for the corresponding property or [NSNull null] if it does not exist
  */
-+ (NSArray<NSString *> *)wdPropertyNames;
++ (NSDictionary<NSString *, id> *)wdProperties;
```

Comment:
not really. This dictionary contains all properties having 'wd' prefixes as keys and the corresponding getter names as values or NSNull is there is no getter defined. This getter is returned to make it compatible with KVO lookup, because objc does not understand property names for such purpose if there is a getter method defined for it.